src/main.cpp

```diff
@@ -539,6 +539,7 @@ int main ( int argc, char** argv )
 #else
 #    if defined( Q_OS_IOS )
     bUseGUI        = true;
+    bIsClient      = true; // Client only - TODO: maybe a switch in interface to change to server?
```

Comment:
Or a separate install for the server?  (MacOS has two bundles.)

Reply:
Yeah why not. However I'm a bit skeptical about iOS server performance (tried it briefly, processing time was high).

Reply:
If a Pi4 can do it, a recent iOS device should be powerful enough, I'd have thought?

Reply:
This hints at a thought I had a while ago: could we add a GUI button to switch into server mode? Probably possible, but this might require some big changes.

Building two bundles for macOS is hacky.

Reply:
We have two GUIs.

We have two main services.

We bundle all four into a single executable and rely on command line options to pick which of the four modes to operate in.

We could have four smaller executables with the shared code in a separate library.  (Again, part of what I'd like to see in the future - restructuring the code like that will help get the GUI and service layers split more neatly.)

Reply:
I would expect no real demand / need to run a server on an iOS device.
I see a strong use for a client here.